pyslurm/pyslurm.pyx

```diff
@@ -2732,20 +2738,22 @@ cdef class job:
         if job_opts.get("tmpdisk"):
             desc.pn_min_tmp_disk = job_opts.get("tmpdisk")
 
-        # TODO: declare and use MAX macro or use python max()?
-#        if job_opts.get("overcommit"):
-#            desc.min_cpus = max(job_opts.get("min_nodes", 1)
-#            desc.overcommit = job_opts.get("overcommit")
-#        elif job_opts.get("cpus_set"):
-#            # TODO: cpus_set
-#            #       check for ntasks and cpus_per_task before multiplying
-#            desc.min_cpus = job_opts.get("ntasks") * job_opts.get("cpus_per_task")
-#        elif job_opts.get("nodes_set") and job_opts.get("min_nodes") == 0:
-#            desc.min_cpus = 0
-#        else:
-#            desc.min_cpus = job_opts.get("ntasks")
-
-        # TODO: ntasks_set, cpus_set
+        if job_opts.get("overcommit"):
```

Comment:
Could we work on some tests for these?

Reply:
`submit_batch_job` does not seem to be tested beyond giving a `wrap` script and a `job_name`. I added simple tests for `ntasks` and `cpus_per_task`...